### ISSUE TITLE
Process Changes 

### ISSUE THREAD

**Issue by Mitzi-Laszlo** [solid/process#74]
Feedback is useful. 

It is also useful for expectations to be set around who is responsible for responding to what feedback and what kind of response the person giving feedback can expect from who. 

The reason I bring this up is that there is a fair bit of conversation around the approved co-created process. 

These conversations are all very theoretical and the current agreement hasn't been implemented yet. 

It is impossible to find a process that caters 100% to everyone, we will need to compromise and I am very grateful to everyone who already has.

As Solid Manager, it is really important to me that everyone feels listened to and at the same time want to avoid getting stuck talking about the process in a legalistic way without maintaining a collaborative productive team spirit.

To avoid this scenario it is important that we draw a line underneath the co-created approved process and try it out for a few months/ years/ until there is am major block to avoid getting stuck in an unproductive never ending conversation about process. 

@RubenVerborgh has raised some concerns on https://github.com/solid/culture/issues/64 and https://github.com/solid/culture/issues/57 and https://github.com/solid/culture/issues/36 and @justinwb has come up with concrete proposal on how to address them. 

Considering it is the first time Solid has such an explicitly written down process and @RubenVerborgh as a member of the MIT academic project and active contributor to Solid you having a good feeling about the process is important I propose we put the three suggestions forward for approval in an exceptional fast track. 

In the current approved process there is an explanation of [how to submit a process proposal](https://github.com/solid/culture/blob/master/README.md#how-to-make-changes).

To avoid having to fast track and debate further in the coming weeks I propose to add another paragraph as follows: 

"The person who starts a process proposal is responsible for actively and publicly sharing it with panels and stakeholders over a minimum time period of one month to gather feedback, which they need to demonstrate incorporating, along with publicly sharing the reasoning for the final result before submitting the process proposal for approval. In the interim the last approved process will be used."

**Comment by RubenVerborgh** [comment]
Reacting here because I was @-mentioned; my intention is to limit myself to this one comment, so I'd appreciate no further @-mentions unless strictly necessary.

Something important has been missed in my opinion. The primary reason we disagreed in #36, @Mitzi-Laszlo, is that you are so far the only person considering the proposal of #36 a process change. That doesn't mean the majority is right; but it does mean that an opinion exists in which #36 is _not_ a process change, and at no point in the discussion, you have acknowledged, considered, or refuted that point; only stated its opposite.

So we had the unfortunate situation where a group of people were arguing for a certain interpretation of the process (that they felt was compatible with the written text), and you repeatedly asking exactly what change we wanted to make to the process (whereas the answer, from our point of view, was "none"). I think it was very well captured by @shaunagm here: https://github.com/solid/culture/issues/36#issuecomment-512266140, in her second question.

The current document, unless I am mistaken, does not appoint anyone yet as a Solid Manager. But even if it did, and if it were you, the process does not give anyone the authority to unilaterally decide whether or not something is a process change. Again, I'm not saying #36 is or isn't, just that there were two views on that, and only one was respected. So we had the unproductive irony where on the one hand the discourse in #36 advocates for a very strict interpretation of the process as being the one true version, whereas on the other hand giving any person the authority to make such an interpretation is not supported by that same spec in the first place.

> It is impossible to find a process that caters 100% to everyone,

I beg to differ, not just as a Solid contributor but also from experience with W3C Working Groups far larger than the Solid group. No disagreements there whatsoever. The Solid process is not going the right way. My best suggestion is to look at what people want to do, and support that by a process; not instill a top-down process and tell people how to do their thing. Yes, I have helped approve #24, but that's only because I did not know it would be implemented in such a strict way that no one is helped by it currently.

So instead of nailing this process perfectly upfront, which is likely an impossibility, let's just treat it with the necessary flexibility (as also suggested by @acoburn in https://github.com/solid/culture/issues/57#issuecomment-513476712). The process is currently hurting rather than helping. It's too heavy, too complex, and serving itself rather than the people it is supposed to serve.

Also a reminder that lack of progress with Solid in the past was not due to a lack of process, but due to a lack of people doing this. So let's not make it harder for those who want to do things, to do them, and to do them in a timely manner.
